src/gen/js/genReduxActions.ts

```diff
@@ -1,6 +1,6 @@
 import { writeFileSync, join, groupOperationsByGroupName, camelToUppercase, getBestResponse } from '../util'
 import { DOC, SP, ST, getDocType, getTSParamType } from './support'
-import { renderParamSignature, renderOperationGroup } from './genOperations'
+import {renderParamSignature, renderOperationGroup, renderOperationDocs} from './genOperations'
```

Comment:
Yeah, right. The block was formatted by the code editor on import.
Should have payed more attention before commit and submit the PR